uint8[2] cnt_data0 
uint8[2] cnt_data1 
uint8[2] cnt_data2 
uint8[2] cnt_data3 
uint16[4] servo